src/components/rangeSelect.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { FC } from "react"
+
+import Select from "./select"
+
+interface Props {
+  name: {
+    min: string
+    max: string
+  }
+  handleChangeMin: (value) => void
+  handleChangeMax: (value) => void
+  value?: {
+    min: number
+    max: number
+  }
+  isValid?: {
+    min: boolean
+    max: boolean
+  }
+  placeholder?: {
+    min: string
+    max: string
+  }
+  required?: boolean
+  options?: Array<{ name: string; value: number | { customValue: number } }>
```

Comment:
Would range select ever accept a custom value? I think we can just roll with:

```suggestion
  options?: Array<{ name: string; value: number }>
```

Reply:
👍 Indeed, In that case we would go for the input version I guess !